update.py

```diff
@@ -70,26 +70,33 @@ def delete_term(term_id):
 
 def create_files(data, term_id):
     print('Writing files...')
+    
+    if not os.path.exists(TERM_DIR_TEMPLATE.format("")):
+      os.mkdir(TERM_DIR_TEMPLATE.format(""))
     term_dir = TERM_DIR_TEMPLATE.format(term_id)
     os.mkdir(term_dir)
 
-    with open('data/terms.json') as terms_file:
+    terms_data = list()
```

Comment:
Is this good practice for Python? I didn't think we needed something like this due to Python scope rules.

Reply:
I did some research, and it looks like we do not, `terms_data = []` is better practice – I will fix